examples/long-term-schedule-transaction.js

```diff
@@ -0,0 +1,191 @@
+        ).execute(client)
+    ).getReceipt(client);
+
+    let info = await new ScheduleInfoQuery()
+        .setScheduleId(scheduleId)
+        .execute(client);
+
+    console.log("Scheduled transaction is not yet executed. ");
+
+    // Step 5: Sign the transaction with the other key
+    let accountBalance = await new AccountBalanceQuery()
+        .setAccountId(alice)
+        .execute(client);
+    console.log(
+        "Alice's account balance before schedule transfer: ",
+        accountBalance.hbars.toString(),
+    );
+
+    console.log("Signing the new scheduled transaction with the 2nd key");
+    await (
+        await (
+            await new ScheduleSignTransaction()
+                .setScheduleId(scheduleId)
+                .freezeWith(client)
+                .sign(privateKey2)
+        ).execute(client)
+    ).getReceipt(client);
+
+    accountBalance = await new AccountBalanceQuery()
+        .setAccountId(alice)
+        .execute(client);
+    console.log(
+        "Alice's account balance after schedule transfer: ",
+        accountBalance.hbars.toString(),
+    );
+
+    info = await new ScheduleInfoQuery()
+        .setScheduleId(scheduleId)
+        .execute(client);
+    console.log(
+        "Scheduled transaction is executed. Executed at: ",
+        info.executed,
+    );
+
+    // Step 6: Schedule another transfer transaction
+    console.log("Creating new scheduled transaction with 20 seconds expiry");
+    const transfer2 = new TransferTransaction()
+        .addHbarTransfer(alice, new Hbar(-1))
+        .addHbarTransfer(client.operatorAccountId, new Hbar(1));
+
+    const scheduleId2 = (
+        await (
+            await transfer2
+                .schedule()
+                .setWaitForExpiry(true)
+                .setExpirationTime(
+                    Timestamp.generate().plusNanos(20 * 1_000_000_000),
+                ) // 10 seconds in milliseconds
+                .execute(client)
+        ).getReceipt(client)
+    ).scheduleId;
+
+    // Step 7: Sign the transaction with one key
+    console.log("Signing the new scheduled transaction with 1 key");
+    await (
+        await (
+            await new ScheduleSignTransaction()
+                .setScheduleId(scheduleId2)
+                .freezeWith(client)
+                .sign(privateKey1)
+        ).execute(client)
+    ).getReceipt(client);
+
+    info = await new ScheduleInfoQuery()
+        .setScheduleId(scheduleId2)
+        .execute(client);
+    console.log("Scheduled transaction is not yet executed.");
+
+    // Step 8: Update the account’s key
+    console.log("Updating Alice's key to be the 1st key");
+    await (
+        await (
+            await (
+                await new AccountUpdateTransaction()
+                    .setAccountId(alice)
+                    .setKey(privateKey1.publicKey)
+                    .freezeWith(client)
+                    .sign(privateKey1)
+            ).sign(privateKey2)
```

Comment:
how are these 2 signs not in one level? :D

Reply:
`sign` returns a promise which I have to await. This makes it so I have to put them on 2 different levels.